#Definition of the input object type
#the id of the object
uint64 id
#the variables of the object
InputVariable[] variables